client-app/shared/static-content/components/slider.vue

```diff
@@ -101,7 +102,7 @@ const modules = [Navigation];
 
 const wrapperClasses = computed(() => {
   return {
-    [props.background]: true,
+    [props.background]: !props.isTransparent,
```

Comment:
Why we can not set `props.background = transparent ?`

Reply:
to keep settings simple for builder.io users